11ty/_includes/layouts/base.njk

```diff
@@ -3,17 +3,20 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
+    <link rel="preconnect" href="https://use.typekit.net/" crossorigin>
+    <link rel="dns-prefetch" href="https://use.typekit.net/">
```

Comment:
`dns-prefetch` alternatives are provided here for browsers that don't understand `preconnect`, but do understand `dns-prefetch`. It's not as big of a boost as `preconnect`, but it's better than nothing.